internal/pkg/ntp/ntp.go

```diff
@@ -357,11 +357,11 @@ func (syncer *Syncer) queryPTP(server string) (*Measurement, error) {
 		return nil, err
 	}
 
-	offset := time.Until(time.Unix(ts.Sec, ts.Nsec))
+	offset := time.Until(time.Unix(ts.Unix()))
```

Comment:
However this one line here might still be a good idea, for readability.

Reply:
yep, that's a good one for sure